ocaml/xapi/xapi_host.ml

```diff
@@ -786,24 +786,35 @@ let local_management_reconfigure ~__context ~interface =
   change_management_interface ~__context interface
     (Record_util.primary_address_type_of_string (Xapi_inventory.lookup Xapi_inventory._management_address_type ~default:"ipv4"))
 
+let assert_not_bond_slave_of ~__context ~self =
```

Comment:
Or in xapi_pif.ml rather than helpers.ml?
Both Xapi_host and Xapi_pool already use Xapi_pif.